include/aspect/mesh_deformation/interface.h

```diff
@@ -237,6 +237,13 @@ namespace aspect
         const std::set<types::boundary_id> &
         get_free_surface_boundary_indicators () const;
 
+        /**
+         * Return the initial topography stored on
+         * the Q1 finite element that describes the mesh geometry.
+         */
+        const LinearAlgebra::Vector &
+        get_initial_topography () const;
```

Comment:
Is this a vector for which only those nodes on the boundary have meaning? If so, can you say so?

Reply:
I have added a few sentences to explain this.